package.json

```diff
@@ -28,7 +28,8 @@
     "lint:fix": "eslint --fix .",
     "prepublishOnly": "npm run clean",
     "proxydirs": "node scripts/proxy-dirs.js",
-    "prepack": "run-s build proxydirs",
+    "github:deps": "npm i --no-save npm-run-all",
+    "prepack": "npm run github:deps && run-s build proxydirs",
```

Comment:
Tested this out and it does seem to work although I don’t quite understand why we don’t need to install cross-env, rollup etc. that are used in the build script.

Builds still fail for me under Node 12 & 14 — `(plugin babel) Error: Cannot find module '@babel/compat-data/corejs3-shipped-proposals'` — but it works under Node 10. (This was the same in #745.)

If it’s as simple as needing npm-run-all, would the following do the same and simplify this?

```suggestion
    "prepack": "npx run-s build proxydirs",
```

Reply:
Interesting.

After it applies this fix, it now complains about `cross-env` not being installed. I think what's happening is that `cross-env` is pulled in by `npm-run-all`.

Reply:
I've reverted the change.